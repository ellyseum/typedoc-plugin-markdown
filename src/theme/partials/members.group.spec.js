describe(`members.group.hbs`, () => {
  const context = {
    children: [{}, {}, {}],
    title: 'Variables',
    kind: 32,
    someChildrenAreExported: true,
    allChildrenAreInherited: false,
    allChildrenArePrivate: false,
    allChildrenAreProtectedOrPrivate: false,
    allChildrenAreExternal: false,
    args: {},
  };
  test('should compile', () => {
    const result = global.compileTemplate('members.group.hbs', context);
    expect(result).toMatchSnapshot();
  });
});
