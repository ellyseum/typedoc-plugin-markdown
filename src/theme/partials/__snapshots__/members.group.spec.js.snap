// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`members.group.hbs should compile 1`] = `
"## Variables

[member]

[member]

[member]"
`;
